.github/workflows/build-latest.yaml

```diff
@@ -0,0 +1,197 @@
+name: build-latest
+on:
+#  workflow_dispatch:
+#    inputs:
+#      distro:
+#        description: Base image distro
+#        required: true
+#        default: debian
+#      imageVersion:
+#        description: Base distro image version/release
+#        required: true
+#        default: bullseye
+#      imageVariant:
+#        description: Base image variant
+#        required: true
+#        default: slim
+  push:
+  pull_request:
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    env:
+      latest-ref: refs/heads/develop
+    strategy:
+      matrix:
+        postgresMajorVersion:
+          - 13
+        postgisMajorVersion:
+          - 3
+        postgisMinorRelease:
+          - 1
+        include:
+          - distro: debian
+            imageVersion: bullseye
+            imageVariant: slim
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Get build cache
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          # Build inputs are:
+          # - Dockerfile recipe
+          # - docker-compose.build.yml
+          # - build args (.example.env)
+          # - base_build directory
+          # - scripts directory
+          key: buildx-${{ hashFiles('Dockerfile', 'docker-compose.build.yml', '.example.env', 'base_build', 'scripts') }}-${{ github.sha }}
+          restore-keys: |
+            buildx-${{ hashFiles('Dockerfile', 'docker-compose.build.yml', '.example.env', 'base_build', 'scripts') }}-
```

Comment:
HashFiles of the build inputs are used to determine if we are doing the same build or not.
If it's the same, just retrieve from cache.